lib/install/config/webpacker.yml

```diff
@@ -0,0 +1,47 @@
+# Note: You must restart bin/webpack-dev-server for changes to take effect
+
+default: &default
+  dev_server:                 # https://webpack.js.org/configuration/dev-server/
+    host: localhost           # Dev server host
+    port: 8080
+    https: false
+
+  paths:                       # ~ = Rails.root
+    source: app/javascript     # ~/:source
+    entry: packs               # ~/:source/:entry
+    output: packs              # ~/public/:output
+    manifest: manifest.json    # ~/public/:output/:manifest
+    config: config/webpack     # ~/:config
```

Comment:
Actually, we should probably flatten *all* of the namespaces. They don't merge into the environments like you'd expect:
```ruby
>> config = YAML.load_file("webpacker.yml")

>> config["default"]["paths"]
=> {"source"=>"app/javascript", "entry"=>"packs", "output"=>"packs", "manifest"=>"manifest.json", "config"=>"config/webpack"}

>> config["test"]["paths"]
=> {"output"=>"packs-test"}
```